src/runtime/Makefile

```diff
@@ -152,9 +148,12 @@ SEVKERNELPARAMS := $(AGENTCONFIGFILEKERNELPARAM) agent.enable_signature_verifica
 SNPKERNELPARAMS := $(AGENTCONFIGFILEKERNELPARAM) agent.enable_signature_verification=false $(AGENT_AA_KBC_PARAMS_SNP)
 KERNELPARAMS += $(ROOTMEASURECONFIG) agent.enable_signature_verification=false $(AGENT_AA_KBC_PARAMS)
 
-FIRMWARETDVFPATH := $(PREFIXDEPS)/share/tdvf/OVMF_CODE.fd
+FIRMWARETDVFPATH := $(PREFIXDEPS)/share/tdvf/OVMF.fd
```

Comment:
This must be `OVMF_CODE.fd`, see: https://github.com/kata-containers/kata-containers/commit/370811b0170f76a6cce4fd151d96a23d8837e0c5

Reply:
Thanks for spotting this